docs/getting-started/detailed-guide.md

```diff
@@ -0,0 +1,389 @@
+
+2. Check if have the correct current kubernetes context. The context should be `kind-claudie`.
+
+    ```bash
+    kubectl config current-context
+    ```
+
+3.  If context is not `kind-claudie`, switch to it:
+
+    ```bash
+    kubectl config use-context kind-claudie
+    ```
+
+4. One of the prerequisites is `cert-manager`, deploy it with the following command:
+
+    ```bash
+    kubectl apply -f https://github.com/cert-manager/cert-manager/releases/download/v1.12.0/cert-manager.yaml
+    ```
+
+5. Download latest Claudie release and unzip it:
+
+    ```bash
+    wget https://github.com/berops/claudie/releases/latest/download/claudie.zip && unzip claudie.zip -d claudie
+    ```
+
+    !!! note "Tip!"  
+        For the initial attempt, it's highly recommended to enable debug logs, especially when creating a large cluster with DNS. This helps identify and resolve any permission issues that may occur across different hyperscalers. Edit `claudie/.env` file, and change `GOLANG_LOG=info` to `GOLANG_LOG=debug` to enable debug logging, for more customization refer to [this table](#claudie-customization).
+
+6. Deploy Claudie using Kustomise plugin:
+    ```bash
+    kubectl apply -k claudie
+    ```
+
+7. Claudie will be deployed into `claudie` namespace, you can view if all pods are running:
+
+    ```bash
+    kubectl get pods -n claudie 
+    ```
+    ```text
+    NAME                           READY   STATUS      RESTARTS        AGE
+    ansibler-5c6c776b75-82c2q      1/1     Running     0               8m10s
+    builder-59f9d44596-n2qzm       1/1     Running     0               8m10s
+    context-box-5d76c89b4d-tb6h4   1/1     Running     1 (6m37s ago)   8m10s
+    create-table-job-jvs9n         0/1     Completed   1               8m10s
+    dynamodb-68777f9787-8wjhs      1/1     Running     0               8m10s
+    frontend-5755b7bc69-5l84h      1/1     Running     0               8m10s
+    kube-eleven-64468cd5bd-qp4d4   1/1     Running     0               8m10s
+    kuber-698c4564c-dhsvg          1/1     Running     0               8m10s
+    make-bucket-job-fb5sp          0/1     Completed   0               8m10s
+    minio-0                        1/1     Running     0               8m10s
+    minio-1                        1/1     Running     0               8m10s
+    minio-2                        1/1     Running     0               8m10s
+    minio-3                        1/1     Running     0               8m10s
+    mongodb-67bf769957-9ct5z       1/1     Running     0               8m10s
+    scheduler-654cbd4b97-qwtbf     1/1     Running     0               8m10s
+    terraformer-fd664b7ff-dd2h7    1/1     Running     0               8m9s
+    ```
+
+    !!! warning "Troubleshoot!" 
```

Comment:
Will be the troubleshooting section part of this PR?

Reply:
This is a great point actually and I'm not sure. I'll create a ticket and make a reference in there that it's not yet complete. Thanks!

Reply:
I've added a new page troubleshooting and a link to it. But the page says that troubleshooting guide is under construction.